packages/client/src/components/Actions/components/Confirmations.js

```diff
@@ -0,0 +1,46 @@
+import Svg from "library/Svg";
+import { useContacts } from "hooks";
+import { getSafeContactName, shortenAddr, shortenName } from "utils";
+
+const Confirmations = ({ confirmations, safeData }) => {
+  const { address: safeAddress } = safeData;
+
+  const { contacts } = useContacts(safeAddress);
```

Comment:
As far as I know contacts are standalone state, it doesn't contain the data of safeOwners. When I'm signing an action of sending a token, I'm expecting to see my name on the confirmation block but I'm only seeing the address. Is this the expected behavior?

Reply:
As I understood from this document https://www.notion.so/Owner-Contacts-Logic-b5c5ee72eb5f4d4887d59c081d510ae4 - we should use saved addresses for logic in modals, send, approve/reject and confirmation
@amyliu6 please let me know your thoughts

Reply:
Thank you! I will update confirmations to contain names of the safe owners.
The information in the modal -> Sent from and Sent to -> i will leave it to be read from the safe contacts, if that's okay.